sdk/rpc-gql-compat/src/mappers/util.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+export function toShortTypeString<T extends string | null | undefined>(type?: T): T {
+	return type?.replace(/0x0+/g, '0x').replace(/,\b/g, ', ') as T;
```

Comment:
I am not clear on when we shorted addresses, this may only be the built in types like 0x2